front/app/api/projects/useProjectById.ts

```diff
@@ -27,7 +27,6 @@ const useProjectById = (id?: string | null, useCache: boolean = true) => {
     queryKey: projectsKeys.item({ id }),
     queryFn: () => fetchProjectById({ id, useCache }),
     enabled: !!id,
-    ...(useCache === false && { staleTime: 0 }),
```

Comment:
Checked with Edwin (who added this) that this was ok to remove. Was causing unnecessary requests to projects that we were supposed to be taken out of the cache (projects are now included in the `admin_publications` request)